me-md/me-md
# Build out basic structure and styling of BiologicalInformation, SelectAge and Location components

## What does this PR do?
This PR creates directories and component files for `BiologicalInformation`, `SelectAge` and `Location`. This PR also updates the App.js snapshot test, and imports the newest components into the App.js file to temporarily render without navigation functionality being implemented.
### Where should the reviewer start?
In the components directory, please look over the `BiologicalInformation`, `Location`, and `SelectAge` files. I have built out the bare bones of the components, but there is not any functionality to them yet. I am not navigating to other screens yet.
#### How should this be manually tested?
When you pull down and open the application, toggle the pages in the render of `App.js` to see what has been added.
#### Have tests been written for all functionality?
No
#### Any background context you want to provide?
I have made some styling adjustments on another branch.
#### What are the relevant tickets?
#26 , #27 , #28 
#### Screenshots (if appropriate)
<img width="535" alt="What is your biological sex" src="https://user-images.githubusercontent.com/31895658/71142876-f297b280-21d5-11ea-99f5-0481cc88c727.png">

#### Additional Notes
I am making a second PR very soon.
